docs/controls/file-pickers/node/select-files.md

```diff
@@ -0,0 +1,102 @@
+---
+author: keco
+ms.author: keco
+ms.date: 10/02/2018
+---
+# Selecting Files with the Microsoft File Browser SDK
+
+The following walk through shows how to integrate the [Microsoft File Browser SDK](https://www.npmjs.com/package/@microsoft/file-browser) with the React application we created during [project setup](index.md).
+
+### Select Files Demo
+
+## 1. Pass a valid Access Token to the SDK
+
+In order to select files with the File Browser SDK, it first needs to successfully fetch items from the Microsoft Graph. Follow [this tutorial](https://developer.microsoft.com/en-us/graph/docs/concepts/auth_overview) to get a valid `access_token`. Once obtained, replace `<access_token>` in the sample code below with
+the valid token so that it is returned as a `Promise` via `getAuthenticationToken`.
+
+```jsx
+class App extends React.Component {
+  public render() {
+    return (
+      <GraphFileBrowser 
+        getAuthenticationToken={this.getAuthenticationToken} />
+    );
+  }
+
+  private getAuthenticationToken() {
+    return new Promise(resolve => resolve('<access_token>'));
+  }
+}
+```
+
+Reload your application with the above change and you should see the File Browser fetch and render items from your root drive.
+
+## 2. Attach success and cancel callbacks
+
+The `GraphFileBrowser` component exposes the following callback props, `onSuccess` and `onCancel`. The `onSuccess` callback is invoked when a User selects files via the default "Select" action button. The `onCancel` callback is invoked when a User cancels a select action via the default "Cancel" action button.
```

Comment:
Is there a reason for capitalizing "user" here? Just curious. :)

Reply:
Personal habit. It looks like we use lower-cased throughout other documentation so I'll change to that.